jujugui/static/gui/src/test/test_env_api.js

```diff
@@ -1293,6 +1294,24 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
     describe('local charm upload support', function() {
 
+      it('uses the correct endpoint when served from juju', function() {
+        window.juju_config = { staticURL: '/static-url'};
```

Comment:
This will need to be reset in an afterEach() because if the test fails then it'll never hit line 1312